Makefile.include

```diff
@@ -452,7 +452,7 @@ $(GENDIR)/%: $(GENDIR)/%.diffupdate | $(GENDIR)
 
 # Start cooja through "make file.csc", where the real target is defined through the csc.
 %.csc %.csc.gz: FORCE
-	$(Q)$(GRADLE) --no-watch-fs --parallel --build-cache -p $(COOJA_PATH) run --args="-quickstart=$(addprefix $(CURDIR)/,$@) -contiki=$(realpath $(CONTIKI)) -logdir=$(CURDIR)"
+	$(Q)$(GRADLE) --no-watch-fs --parallel --build-cache -p $(COOJA_PATH) run --args="--contiki=$(realpath $(CONTIKI)) --logdir=$(CURDIR) $(addprefix $(CURDIR)/,$@)"
```

Comment:
Since we moved to long parameter names - what do you think about skipping the equal sign entirely?

```shell
$(Q)$(GRADLE) --no-watch-fs --parallel --build-cache -p $(COOJA_PATH) run --args="--contiki $(realpath $(CONTIKI)) --logdir $(CURDIR) $(addprefix $(CURDIR)/,$@)"
```

Reply:
The equals sign is how the parameters are described in `--help`, so I think there's a value in Contiki-NG following that.